examples/Atlas/+atlasControllers/QPInputConstantHeight.m

```diff
@@ -22,20 +22,20 @@
   methods
     function obj = QPInputConstantHeight()
       obj.timestamp = 0;
-      obj.zmp_data = struct('A',  [zeros(2),eye(2); zeros(2,4)],... % COM state map % 16 d
-                        'B', [zeros(2); eye(2)],... % COM input map % 4 d
-                        'C', [eye(2),zeros(2)],... % ZMP state-output map % 4 d
-                        'D', -0.89/9.81*eye(2),... % ZMP input-output map % 2 d
-                        'x0', zeros(4,1),... % nominal state % 4 d
-                        'y0', zeros(2,1),... % nominal output % 2 d
-                        'u0', zeros(2,1),... % nominal input % 2 d
-                        'R', zeros(2),... % input LQR cost % 4 d
-                        'Qy', 0.8*eye(2),... % output LQR cost % 4 d
-                        'S', zeros(4),... % cost-to-go terms: x'Sx + x's1 + s2 % 16 d
-                        's1', zeros(4,1),... % 4 d
-                        's1dot', zeros(4,1),... % 4 d
-                        's2', 0,... % 1 d
-                        's2dot', 0); % 1 d
+      obj.zmp_data = struct('A',  [zeros(2),eye(2); zeros(2,4)],... % COM state map 
+                        'B', [zeros(2); eye(2)],... % COM input map
+                        'C', [eye(2),zeros(2)],... % ZMP state-output map
+                        'D', -0.89/9.81*eye(2),... % ZMP input-output map
+                        'x0', zeros(4,1),... % nominal state
+                        'y0', zeros(2,1),... % nominal output
+                        'u0', zeros(2,1),... % nominal input
+                        'R', zeros(2),... % input LQR cost
+                        'Qy', 0.8*eye(2),... % output LQR cost
+                        'S', zeros(4),... % cost-to-go terms: x'Sx + x's1 + s2
+                        's1', zeros(4,1),...
+                        's1dot', zeros(4,1),... 
+                        's2', 0,... 
+                        's2dot', 0);
```

Comment:
Those numbers were my rough count of the number of doubles in the message when I first wrote it, but they're out of date and were often mistaken for the number of dimensions of each variable. The default values do a better job of showing the expected size